go-controller/pkg/libovsdbops/copp.go

```diff
@@ -103,13 +108,31 @@ func CreateDefaultCOPP(nbClient libovsdbclient.Client) (string, error) {
 		m := NewModelClient(nbClient)
 		results, err = m.CreateOrUpdate(opModels...)
 		if err != nil {
-			return coppUUIDResult, fmt.Errorf("error while creating default COPP %+v:%v", controlPlaneProtection, err)
+			return "", fmt.Errorf("error while creating default COPP %+v: %v", controlPlaneProtection, err)
 		}
 		// results will have default meters and copp at last index
 		coppUUIDResult = results[len(results)-1].UUID.GoUUID
 	} else {
 		// copp exists
 		coppUUIDResult = controlPlaneProtection[0].UUID
+
+		// Make sure it has a name as OVN 22.03 requires one.
+		if controlPlaneProtection[0].Name == "" {
+			m := NewModelClient(nbClient)
+			opModels := []OperationModel{
+				{
+					Model: &nbdb.Copp{
+						Name: defaultCOPPName,
```

Comment:
Is changing an index through an update actually allowed? I have no idea.

Reply:
@jcaamano Now with unit tests!